src/eosjs2-jsonrpc.ts

```diff
@@ -31,7 +31,7 @@ export interface Abi {
 /** Return value of `/v1/chain/get_abi` */
 export interface GetAbiResult {
   account_name: string;
-  abi: Abi;
+  abi: Abi | Uint8Array;
```

Comment:
`/v1/chain/get_abi` doesn't return a `Uint8Array`. Define a new interface in `eosjs2-api.ts` instead.